tests/README.md

```diff
@@ -86,6 +86,16 @@ tmt run -v -eWITH_VALGRIND=1
 
 If `valgrind` detects a memory leak in a test, the test will fail, and the logs will be found in the test `data` directory.
 
+## Running integration tests with code coverage
+
+To run integration tests with `code coverage`, set `WITH_COVERAGE` environment variable as follows:
```

Comment:
I'd change the sentence to *To produce code coverage report during integration tests run set `WITH_COVERAGE` environment variable as follows:*